doc/src/modules/evalf.rst

```diff
@@ -343,7 +343,7 @@ digits in a fraction of a second with a simple command:
     >>> f = factorial
     >>> n = Symbol('n', integer=True)
     >>> R = 9801/sqrt(8)/Sum(f(4*n)*(1103+26390*n)/f(n)**4/396**(4*n),
-    ...                         (n, 0, oo)) #doctest: +SKIP
+    ...                         (n, 0, oo))
```

Comment:
Was this skipped because it is slow?

Reply:
@smichr I don't know the real reason why it was skipped, I just checked where the #doctest: +SKIP is used and does it make any difference when they are removed or are they necessary for a specific reason for knowing this I just run tests(checks were OK) and there were no fails so I just thought they can be removed.
*Is there a way to check if this was slow? I am rookie in these things ; (*

Reply:
It takes about 2 seconds to calculate and print on `replit.com`. I suppose part of the question is whether we really need to keep computing this in a doctest since it (should) already be tested in the tests. We can show that is is possible but don't actually need to recompute it every time. @asmeurer  ?